etc/search-core.api.md

```diff
@@ -642,6 +645,20 @@ export interface SearchService {
     verticalSearch(request: VerticalSearchRequest): Promise<VerticalSearchResponse>;
 }
 
+// @public
+export interface ServingConfig {
+    // Warning: (ae-forgotten-export) The symbol "CloudRegion" needs to be exported by the entry point index.d.ts
+    // Warning: (ae-unresolved-inheritdoc-reference) The @inheritDoc reference could not be resolved: The package "@yext/search-core" does not have an export "CloudRegion"
+    //
+    // (undocumented)
+    cloudRegion?: CloudRegion;
+    // Warning: (ae-forgotten-export) The symbol "Environment" needs to be exported by the entry point index.d.ts
+    // Warning: (ae-unresolved-inheritdoc-reference) The @inheritDoc reference could not be resolved: The package "@yext/search-core" does not have an export "Environment"
+    //
+    // (undocumented)

```

Comment:
It looks like CloudRegion and Environment need to be exported again

Reply:
Done